src/block-management/ethrpc-transport-adapter.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
+
+var nextToken = 1;
```

Comment:
> The returned tokens are explicitly strings not numbers.
Why is this?  JS will auto-convert integer keys to strings -- I guess you just prefer to be explicit?

> The fact that they happen to be auto-incremented numbers is an implementation detail and subject to change. A UUI with class prefix would probably be a better system for generating tokens as it would allow the unsubscribe methods to validate that the token provided was reasonable, but I didn't feel writing the code associated with or bringing in the dependencies required to make that happen.

If you intend to change it, IMO the time to do that is now, while all this is still fluid and fresh in your mind.  If you want to use UUIDs specifically, the `uuid` and `uuid-parse` libraries are already dependencies of augur.js -- feel free to bring them into ethrpc.